bean.js

```diff
@@ -101,13 +101,13 @@
             // a whitelist of properties (for different event types) tells us what to check for and copy
         var commonProps  = str2arr('altKey attrChange attrName bubbles cancelable ctrlKey currentTarget ' +
               'detail eventPhase getModifierState isTrusted metaKey relatedNode relatedTarget shiftKey '  +
-              'srcElement target timeStamp type view which propertyName')
+              'srcElement target timeStamp type view which propertyName path')
```

Comment:
No.
Those where added in a previous PR #116. I just made a new build. As those new properties where added in the source but where not yet compiled to the build.